quinn/src/connection.rs

```diff
@@ -19,15 +19,18 @@ use tokio_timer::Delay;
 use crate::streams::{NewStream, RecvStream, SendStream, WriteError};
 use crate::{ConnectionEvent, EndpointEvent};
 
-/// Connecting future
+/// In-progress connection attempt future
+///
+/// Be sure to spawn the `ConnectionDriver` when complete.
 pub struct Connecting(Option<ConnectionDriver>);
 
 impl Connecting {
     pub(crate) fn new(conn: ConnectionRef) -> Self {
         Self(Some(ConnectionDriver(conn)))
     }
 
-    /// Convert into a 0-RTT or 0.5-RTT connection at the cost of weakened security
+    /// Convert into a 0-RTT or 0.5-RTT connection at the cost of weakened security. Be sure to
```

Comment:
Hmm, so I think it should go after the newline here too, and in the third hunk as well. Otherwise the rustdocs will look weird.

Reply:
Current rustdoc doesn't seem to treat the first line of method documentation specially, unless I'm missing something.

Reply:
Ah, you're right, it doesn't matter for methods!